.github/ISSUE_TEMPLATE/71_org_editor-recruit_reviewers.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "[Organizer -> Editor] Recruit Reviewers"
+about: Checklist for Associate Editor to fill out prior to review
+title: "[PRE-REVIEW] Recruit Reviewers"
+labels: pre-review
+assignees: ''
+---
+
+<!--
+## DO NOT EDIT THIS FILE OUTSIDE OF THE journalovi/jovi-workflows REPOSITORY
+##
+## This file is automatically updated in all repositories within the journalovi
+## Github organization whenever the version in journalovi/jovi-workflows is
+## changed, so any other edits will be overwritten. To update this file, make
+## a commit or pull request at https://github.com/journalovi/jovi-workflows
+-->
+
+### Instructions for Editor
+- Send out review requests using the [reviewer invitation email template](https://docs.google.com/document/d/15ttXvQg7g5JEk2Yzfd9eXSN6oIpc_A61qbuwygthuG0/edit?usp=sharing).
+  - Replace "LINK TO REVIEW ISSUE TEMPLATE" in the email with [this link](new?assignees=&labels=review&projects=&template=00_reviewer-review.yml&title=%5BREVIEW%5D+%3Ctitle%3E) (Right click to copy it.)
+- When a reviewer agrees to review, complete the corresponding steps for that reviewer below.
+- Close this issue when all reviewers have been recruited.
+
+### Recruit reviewers:
+- [ ] Reviewer 1 recruited.
+  - [ ] [Review Request Issue](new?assignees=&labels=pre-review&projects=&template=20_editor_reviewer-write_review.md&title=%5BPRE-REVIEW%5D+Review+Request) created and assigned to them.
+  - [ ] Organizer notified on Slack (or tagged here).
```

Comment:
I'm unsure if this step is necessary given that the OC is assigned to the repo, and that this issue serves as a status tracker for reviewers.

Reply:
The OC won't be notified of boxed being checked, so tagging is still needed for situational awareness.

Reply:
OK. Just to be clear, by "tagging" you mean: adding a comment with @-mention the OC, right? If so, may be we should write that. (Especially because "tag" has another meaning in git context.)